### ISSUE TITLE
No module named 'main'

### ISSUE THREAD

**Issue by ilanreuben** [pydn/ComfyUI-to-Python-Extension#66]
After running comfyui_to_python.py successfully, this is what happens when I run the resulting script:

$ python workflow_api.py 
Traceback (most recent call last):
  File "/home/ilana/ai/comfy/ComfyUI-to-Python-Extension/workflow_api.py", line 83, in <module>
    add_extra_model_paths()
  File "/home/ilana/ai/comfy/ComfyUI-to-Python-Extension/workflow_api.py", line 72, in add_extra_model_paths
    from main import load_extra_path_config
ModuleNotFoundError: No module named 'main'

(The original workflow_api.json works.)

Any suggestions?

**Comment by ilanreuben** [comment]
OK, simple enough -- workflow_api.py needs to be in the main comfy directory

**Comment by chronoNote** [comment]
Quici question @ilanreuben  do you mind?
I am not entierly sure how this works, I installed it with manager. Then activated developer option. then saved a workflow with save as a script.
Then to run the script I opened a terminal , and did "py name of my script" but it does nto work because it asks me for different libraries (indeed this is base python and not an env)
How did you make it work step by step please?

**Comment by pydn** [comment]
Try moving the script to your ComfyUI directory and be sure you are using the same environment you use when running ComfyUI when you run your script.

**Comment by chronoNote** [comment]
I did not believe I would get an answer I was already writing a new issue then i saw this.
Ok the problem is that comfy comes with an embedded python that has all the python necessary libraires right? So that quick installation tool offered by comfyUI would not work with this extension? Since the python is not exaclty inside a venv or conda right?
And indeed when I tried to launch the py file created with this extension it asked for libraries like in this image:
![image](https://github.com/user-attachments/assets/a2127189-ef2b-4ab5-8cfd-26caa3422352)

Did I do something wrong perhaps?
